### DISCUSSION TITLE
Regarding increasing size of scrollpane's upscroll icon and downscroll icon.

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2786] [Guides & Tutorials]
Sorry for asking the basic need, but I didn't used the component till yet. But it is now required for my app.

My stage size is 3840 x 2160. If I used the scrollpane component and add some data which enable the upscroll and downscroll. But when publish the swf, it is very complicated to scrollup and scrolldown. Even I cannot see the bar. Is there any way to increase the size of scrollpan's upscroll and downscroll and even the width of the bar of scroller.

I used the below lines but failed :
```
scrollPaneMovieClipName.verticalScrollBar.downArrow.scaleX = 7;   //OR

scrollPaneMovieClipName.verticalScrollBar.downArrow.height = 100
scrollPaneMovieClipName.verticalScrollBar.downArrow.width= 100
```

Thanks.

**Comment by balaedit** [comment]
Is there anyone who can help me in this discussion of scrollpane.

**Reply by ajwfrost** [reply]
This is a bit outside of our normal expertise, but can I check if you're using Adobe Animate for this? I think the scrollpane assets would be imported into your library, so it may be possible to update things there?

If you can provide a simple/basic example using FLA/AS code, then it may be possible to check this further. But we don't have any of the source for these components, they're completely separate from the Flash/AIR runtimes, so there's a limit to what we'd be able to do.

thanks

**Reply by balaedit** [reply]
@ajwfrost Sorry for delay, was busy on other task as priority switch. Back to point the sample file is below. Needs to increase the size of scroller of vertical which is too small. Please do this needful. Thanks.

[scrollPane.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/13612530/scrollPane.zip)

**Reply by balaedit** [reply]
@ajwfrost @PrimaryFeather Any Solution on this.... Please help...

**Reply by balaedit** [reply]
@ajwfrost I had created the custom movieclip scroller to resolve the issue. I removed the scrollpane usage from the app as we cannot increase the size of the scrollpane scrollbar or no idea how to increase the size of the scrollpane scrollbar. Thanks.
